builtin/providers/google/resource_google_project.go

```diff
@@ -172,6 +178,21 @@ func resourceGoogleProjectCreate(d *schema.ResourceData, meta interface{}) error
 		}
 	}
 
+	// Set the billing account
+	if v, ok := d.GetOk("billing_account"); ok {
+		ba := cloudbilling.ProjectBillingInfo{
+			BillingAccountName: "billingAccounts/" + v.(string),
+		}
+		_, err = config.clientBilling.Projects.UpdateBillingInfo("projects/"+pid, &ba).Do()
```

Comment:
nit: even though it's a teeny tiny thing, since you do "projects/" + pid a few times I wouldn't mind seeing that in a function